README.md

```diff
@@ -295,18 +295,19 @@ If you have received the package and only want to build and run the docker conta
 To run it in a docker container with access to your local network interface, run:
 
 ```bash
-docker run                                                                       \
-    --rm                                                                         \
-    --network     host                                                           \
-    --hostname    signing-service                                                \
-    --name        signing-service                                                \
-    --volume      < path to file >/contract.key:/etc/contract.key:ro             \
-    --restart     on-failure                                                     \
-    concent-signing-service                                                      \
-        --host                      concent.golem.network                        \
-        --port                      9055                                         \
-        --dsn                       https://1234:5678@talkback.golem.network/555 \
-        --ethereum-private-key-path /etc/contract.key
+docker run                                                                               \
+    --rm                                                                                 \
+    --network     host                                                                   \
+    --hostname    signing-service                                                        \
+    --name        signing-service                                                        \
+    --restart     on-failure                                                             \
+    concent-signing-service                                                              \
+        concent.golem.network                              #host                         \
```

Comment:
I think that
```
--host                      concent.golem.network
```

was better than having a comment like that.

Reply:
The `--host` flag not exist.

Reply:
OK